### ISSUE TITLE
Segmentation Fault while feature extracation

### ISSUE THREAD

**Issue by yash1994** [google/mediapipe#423]
I've followed all the installation steps and successfully ran the Hello World desktop example. But when i run the following command for feature extraction, i am getting segmentation fault (core dumped) without any error logs.
```
bazel build -c opt \
  --define MEDIAPIPE_DISABLE_GPU=1 --define no_aws_support=true \
  mediapipe/examples/desktop/youtube8m:extract_yt8m_features

GLOG_logtostderr=1 bazel-bin/mediapipe/examples/desktop/youtube8m/extract_yt8m_features \
  --calculator_graph_config_file=mediapipe/graphs/youtube8m/feature_extraction.pbtxt \
  --input_side_packets=input_sequence_example=/tmp/mediapipe/metadata.pb  \
  --output_side_packets=output_sequence_example=/tmp/mediapipe/features.pb
```
![Screenshot from 2020-01-30 13-53-30](https://user-images.githubusercontent.com/13917659/73432155-271b9400-4368-11ea-873a-05b2e84a8617.png)

Also tried with `--copt -DMESA_EGL_NO_X11_HEADERS` GPU support but facing the same issue.
Configuration:
OS: Ubuntu 18.04.3 LTS
GPU: NVIDIA GeForce GTX 1080 TI (10 GB)
cuda: 10.0
Cudnn: 7.4
Tensorflow: 1.15.2 (built from source), also tried with 1.14.0 (same issue)
Bazel: 1.0.0
gcc: 7.4.0
g++: 7.4.0
opencv: 4.2.0-dev
ffmpeg: 4.2.2

**Comment by jiuqiant** [comment]
Do you try to extract features from your own video or a public video that we can download for debugging? Also, what's the video format?
BTW, MediaPipe prefers OpenCV@3 rather than OpenCV@4.

**Comment by yash1994** [comment]
Thanks, @jiuqiant for your response. I've downloaded a few videos from YouTube in .mp4 format and for all the videos I am getting the same error. Also, I have installed OpenCV using the script `setup_opencv.sh` provided in this repository. shall I install OpenCV@3?
Providing google drive link for the test video below:
[test_video.mp4](https://drive.google.com/open?id=1Ql119t5X0vQKp_E3ZS9HWGpAipwdYu0m)

Currently, I am using [this] (https://github.com/google/youtube-8m) for the task and it's working fine on all the videos.

**Another question**: Is there any way I get pre-trained weight for ensemble model (Deep bag-of-frames, LSTM) trained on YouTube-8M because i don't have that kind of infrastructure available which requires train those models.
